test/e2e/docker/client.go

```diff
@@ -88,7 +91,7 @@ func RunContainerInBackground(
 		)
 	}
 	defer out.Close()
-	_, _ = io.Copy(io.Discard, out)
```

Comment:
What do you think about moving this into err statement and not outputting on success? Its pretty messy right now
```
• [22.025 seconds]
Successful pull image that only exists in mirror using mirror address [mirror]
/Users/dkoshkin/src/github.com/mesosphere/kubelet-image-credential-provider-shim/test/e2e/suites/mirror/mirror_test.go:135

  Captured StdOut/StdErr Output >>
  {"status":"Pulling from library/nginx","id":"stable"}
  {"status":"Already exists","progressDetail":{},"id":"a603fa5e3b41"}
  {"status":"Pulling fs layer","progressDetail":{},"id":"50b97857b95c"}
  {"status":"Pulling fs layer","progressDetail":{},"id":"0edfe97a837a"}
  {"status":"Pulling fs layer","progressDetail":{},"id":"f95a257c65b6"}
  {"status":"Pulling fs layer","progressDetail":{},"id":"0bf5e07cb0af"}
  {"status":"Pulling fs layer","progressDetail":{},"id":"f3c8f37c59f0"}
  {"status":"Waiting","progressDetail":{},"id":"f3c8f37c59f0"}
  {"status":"Waiting","progressDetail":{},"id":"0bf5e07cb0af"}
  {"status":"Downloading","progressDetail":{"current":624,"total":624},"progress":"[==================================================\u003e]     624B/624B","id":"0edfe97a837a"}
  {"status":"Verifying Checksum","progressDetail":{},"id":"0edfe97a837a"}
  {"status":"Download complete","progressDetail":{},"id":"0edfe97a837a"}
  {"status":"Downloading","progressDetail":{"current":758,"total":957},"progress":"[=======================================\u003e           ]     758B/957B","id":"f95a257c65b6"}
  {"status":"Downloading","progressDetail":{"current":957,"total":957},"progress":"[==================================================\u003e]     957B/957B","id":"f95a257c65b6"}
  {"status":"Verifying Checksum","progressDetail":{},"id":"f95a257c65b6"}
  ...
```

Reply:
Yeah sounds good to me.

Reply:
nvm, I tried that but `out` is nil on error.

Reply:
Ah yeah I think we need to parse the output for errors

Reply:
I think this is why it's failing sometimes, the err is nil but the output contains an error that we could detect and fail on

Reply:
The errors could be related to special characters in the password https://github.com/docker/for-linux/issues/1085